### PR TITLE
binary-search: Add test version string

### LINKED ISSUE #989: binary-search: check test version and update if necessary
As identified in #784, some of the tests currently don't reference the version number for the `canonical-data.json` that they are based on, `binary-search` is one of them. In order to help with the ongoing maintenance of this track, we're in the process of checking the tests against the current version of the `canonical-data.json` - if they are up-to-date, then you can just add the version string, but if they are out-of-date, then the tests will need updating.

Tests should typically be ordered according to their order in the canonical data, since this makes it easier to check and update them in future. If you think that there's a better order, please let us know your reasoning in your PR.

If there are some additional tests that currently exist in this track but which aren't in the canonical data, these can be kept if you think that they are useful. They should be placed at the bottom of the test cases, and it would also be super-helpful if you could add a comment before them saying `# Additional tests for this track` to make it clear that they aren't part of the canonical data.

The version string should be placed just before the first test class and followed by a blank line. For this exercise, the version string is:
```
# Tests adapted from `problem-specifications//canonical-data.json` @ v1.0.0
```

**Test suite:** [exercism/python/exercises/binary-search/binary_search_test.py](https://github.com/exercism/python/blob/master/exercises/binary-search/binary_search_test.py)
**Canonical data:** [exercism/problem-specifications/exercises/binary-search/canonical-data.json](https://github.com/exercism/problem-specifications/blob/master/exercises/binary-search/canonical-data.json)

### PR DESCRIPTION
**Pull request by CSJedi**
Hi, I checked data in binary_search_test.py with data in canonical-data.json

resolves #989

### CODE REVIEW

File: exercises/binary-search/binary_search_test.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 from binary_search import binary_search
 
+# Tests adapted from `problem-specifications//canonical-data.json` @ v1.1.0
```
Comment (N-Parsons) [inline exercises/binary-search/binary_search_test.py]: Looks like the version number is wrong - the canonical data is version `1.0.0`.

This like also need to have an extra blank line before it in order to pass the `flake8` tests for compliance with PEP8.

**Comment by N-Parsons** [comment]
Hi @CSJedi, could you include `resolves #989` in the PR description? [Magic words](https://help.github.com/articles/closing-issues-using-keywords/) help us to keep track of issues.

It would also be helpful in future PRs to have the exercise name at the start of the PR title - it makes it easier for maintainers to see at a glance.

File: exercises/binary-search/binary_search_test.py (hunk 1 above)
Comment (CSJedi) [reply exercises/binary-search/binary_search_test.py]: Hi, so if I change nuber of version , all will pass good?

**Comment by CSJedi** [comment]
I added resolves #989 to description of this issue. Did I do this right? You mean that on "resolves #989"

**Comment by CSJedi** [comment]
Do you need help in checking another files?

**Comment by CSJedi** [comment]
Do I do this check right?

**Comment by m-a-ge** [comment]
@CSJedi what's the deal with all of this renaming stuff?

**Comment by cmccandless** [comment]
@CSJedi The `hacktoberfest` label is not required for [Hacktoberfest](https://hacktoberfest.digitalocean.com/) credit.

**Comment by N-Parsons** [comment]
Hi @CSJedi, thanks for fixing the version number. Looks like everything is good now, so I'll merge it.

I've updated the description and title of your PR for you, to show you what I meant. Using this format for the title helps us to easily see what the PR does, and the [magic words](https://help.github.com/articles/closing-issues-using-keywords/) in the description (appears as your first comment here) mean that the corresponding issue will automatically be closed when the PR is merged.

**Pull request merged**